resource-definitions/terraform-driver/co-provision/aws-policy-co-provision.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: entity.humanitec.io/v1b1
+kind: Definition
+metadata:
+  id: aws-policy-co-provision
+entity:
+  type: aws-policy
+  driver_type: humanitec/terraform
+  # Use the credentials injected via the driver_account to set variables as expected by your Terraform code
+  driver_account: aws
```

Comment:
Out of curiosity: why don't we use the account reference here?

Reply:
Understood. I thought it might make sense in this case to explicitly use the same account for the bucket and the policy, as the policy is already depending on the bucket, but it's just a thought of mine, definitely not a blocker in any sense.